demo-config.json

```
@@ -27,7 +27,8 @@
         "website-id": 1,
         "website-name": "Main Website"
       },
-      "commerce-assets-enabled": true
+      "commerce-assets-enabled": true,
+      "commerce-assets-quality": 95
```

"commerce-assets-quality" doesn't do anything, does it? Just remove.

```suggestion
      "commerce-assets-enabled": true
```